IFC4x3/Properties/i/InstallationMethod_0kHNh$JQX6Ug6mo88oYYDW/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="InstallationMethod_0kHNhJQX6Ug6mo88oYYDW" Name="InstallationMethod" UniqueId="2e457aff-4da8-467a-a1b0-c882328a2360" PrimaryDataType="IfcLabel" />
```

Comment:
This is an existing property: `InstallationMethod_1y3E9FdGDDcuFQAiNae6DC`

> Method of installation of cable/conductor. Installation methods are typically defined by reference in standards such as IEC 60364-5-52, table 52A-1 or BS7671 Appendix 4 Table 4A1 etc. Selection of the value to be used should be determined from such a standard according to local usage.

suggest modifying the label and reusing this one.